x/accesscontrol/types/access_operations.go

```diff
@@ -0,0 +1,63 @@
+package types
+
+import (
+	acltypes "github.com/cosmos/cosmos-sdk/types/accesscontrol"
+)
+
+type AccessOperationSet struct {
```

Comment:
curious, why do we want a struct containing a map instead just having like

```
type AccessOperationSet map[aclypes.AccessOperation]struct{}
```

Reply:
to hide the underlying map so that callers can't do things like `del` on it directly